mmfunctions

```diff
@@ -126,8 +126,9 @@ _premis_event_list(){
 
     if [ "$eventDetail" = "ingestfile" ] ; then
         eventType="creation"
-        METADIR="${OUTDIR_INGESTFILE}/${MEDIAID}/metadata/fileMeta/objects"
-        MEDIAINFO=$(cat "${METADIR}/${MEDIA_ID}_mediainfo.xml")
+        METADIR="${AIP_STORAGE}/${MEDIAID}/metadata"
+        MEDIAINFO=$(cat "${METADIR}/fileMeta/objects/${MEDIA_ID}_mediainfo.xml")
```

Comment:
hmmm, I think I prefer to leave fileMeta as the turf of `makemetadata`. Can you move this up to `$METADIR`? Also the value of `$METADIR` may also be set in `_find_input`.